src/net_processing.cpp

```diff
@@ -2081,6 +2086,17 @@ void PeerManagerImpl::StartScheduledTasks(CScheduler& scheduler)
     scheduler.scheduleFromNow([&] { ReattemptInitialBroadcast(scheduler); }, delta);
 }
 
+void PeerManagerImpl::UpdatedBlockTipSync(const CBlockIndex* pindexNew)
+{
+    AssertLockNotHeld(m_mempool.cs);
+    LOCK(m_tx_download_mutex);
```

Comment:
Is clearing the filters is necessary in IBD?

Reply:
gating on whether ibd now